hummingbot/connector/parrot.py

```diff
@@ -55,44 +60,65 @@ async def get_campaign_summary(exchange: str, trading_pairs: List[str] = []) ->
     return results
 
 
-async def get_market_snapshots(market_id: int):
+async def get_market_snapshots(market_id: int, timestamp: int = 1657747860000):
     async with aiohttp.ClientSession() as client:
-        url = f"{PARROT_MINER_BASE_URL}market_snapshots/{market_id}?aggregate=1m"
+        url = f"{PARROT_MINER_BASE_URL}user/single_snapshot?client_id=00000000000000000000000000000000000000000&market_id={market_id}&timestamp={timestamp}&aggregate_period=60"
```

Comment:
why the client id is hardcoded an 00000000000000000000000000000000000000000
and also, why we are aggregating the period in 60 instead of 1m?

Reply:
This seemed to be the way to call a 'user' API method in a public way, otherwise you need to have an authenticated client_id
This seemed to be the way to pass 1m
Note that I do not have (or don't know where to find) the documentation for this API, so all these were gathered from looking at the commands with Brave -> 'Developer tool' when using the miner.hummingbot.io site with my account